MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/plugins/LocationComponentActivity.java

```diff
@@ -1,29 +1,32 @@
 package com.mapbox.mapboxandroiddemo.examples.plugins;
 
-// #-code-snippet: location-plugin-activity full-java
+// #-code-snippet: location-component-activity full-java
```

Comment:
@langsmith does this change need to be reflected in the `android-docs` repo?

Reply:
you can actually remove that `code-snippet` line from the top and bottom of the file. https://github.com/mapbox/mapbox-android-demo/pull/870 got rid of them for all other examples

Reply:
and no, no changes are needed in `/android-docs`